sysutils/ntpstat/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        darkhelmet ntpstat 86e0a68df62e7f6b6c5ee97ee9d15b5e28a4b359
+
+name                ntpstat
+version             0.2
+categories          sysutils net
+platforms           darwin
+license             GPL-2
+maintainers         {hotmail.com:amtor @RobK88} \
+                    openmaintainer
+
+description         ${name} shows the network time protocol (ntp) status
+
+long_description    ${name} shows the network time protocol (ntp) status.\
+                    ${name}  will  report the synchronisation state of the \
+                    NTP daemon running on the local machine.  If the local \
+                    system is found to be synchronised to a reference time \
+                    source, ${name} will report the approximate time accuracy.
+
+homepage            https://github.com/darkhelmet/ntpstat
+
+checksums           rmd160  7481a76c3ac765513113b8db4e0eee3479a65413 \
+                    sha256  f1430f105902f20b4c7c3abdd3cbd69162b0e9275e6967bb620d546572bed8de \
+                    size    3962
+
+use_configure        no
```

Comment:
since you don't use a configure phase, you are responsible to make sure that the port is [UsingTheRightCompiler](https://trac.macports.org/wiki/UsingTheRightCompiler) and MacPorts' flags, that is currently not the case. In many cases the `makefile` PG can be helpful for this.